ComponentKit/Core/CKDimension.mm

```diff
@@ -131,12 +131,11 @@ _Range intersect(const _Range &other) const
 }
 size_t CKSizeRange::hash() const
 {
-  std::hash<CGFloat> hasher;
   NSUInteger subhashes[] = {
-    hasher(min.width),
-    hasher(min.height),
-    hasher(max.width),
-    hasher(max.height)
+      NSUInteger(min.width),
+      NSUInteger(min.height),
+      NSUInteger(max.width),
+      NSUInteger(max.height)
```

Comment:
Are you sure these changes are necessary? I agree the other one probably is, but `hasher` returns a `std::size_t` which should promote to `NSUInteger` without a problem.

Casting `CGFloat` to `NSUInteger` doesn't sit well with me (even though it probably works).